frontend/src/components/common/checkboxes/CheckBox.css

```diff
@@ -0,0 +1,55 @@
+/* Used only for Quiz component */
```

Comment:
I'd prefer to put this inline in the component, but if it's easier to keep it this way, would you be able to make the class names themselves a little more checkbox-specific? That way it's clear what they're for from reading the HTML.